field.go

```diff
@@ -431,6 +432,8 @@ func (f *Field) Open() error {
 	return nil
 }
 
+var fieldQueue = make(chan struct{}, 16)
```

Comment:
should we base this on NumCPU?

Reply:
i thought about it some, but didn't convince myself in either direction. it can be a future improvement.